main/src/test/java/org/mobilitydata/gtfsvalidator/validator/DuplicateFareRuleZoneIdFieldsValidatorTest.java

```diff
@@ -93,7 +93,10 @@ public void noDuplicateRowWithEmptyValuesValueShouldNotGenerateNotice() {
             ImmutableList.of(
                 createFareRule(3, "fare id value", "route id", null, "by id", "to id"),
                 createFareRule(
-                    99, "other fare id value", "route id", "other from id", null, "to id")));
+                    99, "other fare id value", "route id", "other from id", null, "to id"),
+                createFareRule(
+                    99, "other fare id value", null, "other from id", null, "to id")
```

Comment:
Is it normal that the `CsvRowNumber` is the same?

Reply:
Changing this, thanks!

Reply:
Done!